docs/immortal-empires/vampire-counts.md

```diff
@@ -0,0 +1,33 @@
+# Vampire Counts
+
+## Mannfred von Carstein: The Drakenhof Conclave
+
+* **Route I - The Restless Dunes**
+  * The shifting sands of Nehekhara hide many long lost tombs filled with powerful magical artifacts that Mannfred desires. They also hide many foes who would see him undone. They will find only death and an unlife of eternal servitude in these dunes.
+  * Destroy factions:
+    * Khemri, Cult of Sigmar, Chevaliers de Lyonesse
+  * Control provinces:
+    * Shifting Sands, Land of the Dervishes, The Cracked Land, Land of the Dead, Coast of Araby, Land of Assassins, Great Desert of Araby
+  * **Reward**:
+    * _The Ossified Portal_: Creates a dilemma
```

Comment:
I think `Creates a dilemma` is not descriptive enough. I would say something like "Triggers a dilemma that allows Mannfred to teleport into Sylvannia"

Reply:
It's placeholder text because I don't know when the portal stuff is going to be scripted so I didn't bother fleshing out the flavour text yet.